src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/recallnotification/RecallNotificationContextFactory.kt

```diff
@@ -60,8 +59,7 @@ data class RecallNotificationContext(
   fun getRevocationOrderContext(): RevocationOrderContext {
     return RevocationOrderContext(
       recall.recallId(),
-      FirstAndMiddleNames(FirstName(prisoner.firstName!!), prisoner.middleNames?.let { MiddleNames(it) }),
-      LastName(prisoner.lastName!!),
+      PersonName(FirstName(prisoner.firstName!!), prisoner.middleNames?.let { MiddleNames(it) }, LastName(prisoner.lastName!!)),
```

Comment:
If we add a secondary constructor to PersonName:
`
  constructor(firstName: String, middleNames: String? = null, lastName: String) :
    this(FirstName(firstName), middleNames?.let { MiddleNames(it) }, LastName(lastName))
`
then all the (17?) calls like this one simplify to e.g.:
`      PersonName(prisoner.firstName!!, prisoner.middleNames, prisoner.lastName!!),`